index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+      <h1 id="title">This is Zakariya & Usama Tribute Page</h1>
+      <div id="img-div">
+        <img class="center" id="image" src="#" alt="Zack&Usama">
+        <div id="img-caption">Hi, we are Zakariya & Usama</div>
+        <div class="center" id="tribute-info">Hi Microverse</div>
+        <br>
+        <p id="img-caption">Usama The Lion & Zakariya The Programmer</p>
+        <br>
+      </div>
+      <h3> Usama and Zakariya are programming 
+      <a id="tribute-link" href="www.google.com" target="_blank"> google </a>
+    </h3>
+      </main>
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
Kindly avoid using comments that are meaningless in the code as it makes your code not clean.